UITests/Global.swift

```diff
@@ -219,7 +219,7 @@ class BrowserUtils {
         case History = "Browsing History"
         case Cache = "Cache"
         case OfflineData = "Offline Website Data"
-        case Cookies = "Cookies"
+        case Cookies = "Cookies and site Data"
```

Comment:
```suggestion
        case Cookies = "Cookies and Site Data"
```

(capitalize `site`)

This is just in the UI tests, which we don't even use, but may as well keep it updated 😄